install_ant-media-server.sh

```diff
@@ -161,7 +161,7 @@ distro () {
         exit 1
       fi
 
-      if [[ $VERSION_ID != 18.04 ]] && [[ $VERSION_ID != 20.04 ]] && [[ $VERSION_ID != 8* ]]; then
+      if [[ $VERSION_ID != 18.04 ]] && [[ $VERSION_ID != 20.04 ]] && [[ $VERSION_ID != 22.04 ]] && [[ $VERSION_ID != 8* ]]; then
```

Comment:
Firstly, It's also good to support Ubuntu 21.04, 21.10, and 22.10 as well.
Secondly, Red Hat and Alma Linux 9 support will be also good.